internal/target/tcp/repository_tcp_target_test.go

```diff
@@ -251,6 +252,31 @@ func TestRepository_CreateTarget(t *testing.T) {
 			// add an appropriate assert.
 		})
 	}
+
+	t.Run("create with aliases", func(t *testing.T) {
```

Comment:
Can you add error cases where two of the same aliases are added, and an invalid alias?